AuthoringToolTest/PresentationLogic/ModalDialog/ModalDialogInputFieldsFactoryUt.cs

```diff
@@ -9,6 +15,277 @@ namespace AuthoringToolTest.PresentationLogic.ModalDialog;
 
 public class ModalDialogInputFieldsFactoryUt
 {
+    [Test]
+    public void GetCreateLearningElementInputFields_Test()
```

Comment:
MethodenName_Test sagt leider wenig darüber aus, was genau getestet wird und was passieren soll. Wenn der Test in der Zukunft rot wird, muss der/die Entwickler/in erstmal nachforschen, was der Test jetzt eigentlich macht. Daher bitte an unser Schema aus den Code Guidelines (https://wiki.projekt-adler.eu/de/AMG/csharp-code-guidelines) halten: UnitOfWorkName_Szenario_ErwartetesVerhalten

Reply:
`public void GetCreateLearningElementInputFields_ValidDragAndDrop_ReturnsCorrectInputFields(string fileExtensionForTest)`